blueprints/codes/latex_formula.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class LatexFormula:
+    return_symbol: str
+    result: str
+    equation: str = ""
+    numeric_equation: str = ""
+
+    @property
+    def complete(self) -> str:
+        all_sub_equations = [self.return_symbol, self.equation, self.numeric_equation, self.result]
+        return " = ".join([eq for eq in all_sub_equations if eq != ""])
```

Comment:
this will (probably) not work for inequalities?
+ include formula label

Reply:
I have added the formula label as comparison_operator_label with = as default.